### DISCUSSION TITLE
Inconsistent rank after replay

### DISCUSSION THREAD

**Discussion by szmarczak** [ppy/osu#14913] [Q&A] [answered]
I played a map, got a C, watched the replay and it showed B. I'll share more info when I can reproduce.

**Comment by integerrr** [comment, marked as answer]
#14573

**Reply by szmarczak** [reply]
Thanks! I used the search but couldn't find the issue.

**Reply by integerrr** [reply]
https://github.com/ppy/osu/issues?q=is%3Aissue+is%3Aopen+replay+better

**Reply by szmarczak** [reply]
I used these keywords FYI:

https://github.com/ppy/osu/issues?q=is%3Aissue+is%3Aopen+replay+rank+sort%3Aupdated-desc+
